util/build.gradle

```diff
@@ -16,27 +16,28 @@
 apply plugin: 'java-library'
 
 jar {
-  baseName 'besu-util'
+  archiveBaseName = 'besu-util'
   manifest {
     attributes(
-      'Specification-Title': baseName,
+      'Specification-Title': archiveBaseName,
       'Specification-Version': project.version,
-      'Implementation-Title': baseName,
+      'Implementation-Title': archiveBaseName,
       'Implementation-Version': calculateVersion()
       )
   }
 }
 
 dependencies {
+  api 'org.apache.logging.log4j:log4j-api'
```

Comment:
Interesting, is the reason for promoting from `implementation` to `api` due to a Log4j-api class being in one of the signatures of the `util` classes, or Gradle 7.0?

Reply:
They are getting rid of the `runtime` dependency type, and `runtimeOnly` excludes the dependent types from compilation visibility.  i.e. `runtime` was `api` + `runtimeOnly`.  This seemed cleaner than adding log4j2 dependencies in 10 or so new build files.

Reply:
> This seemed cleaner than adding log4j2 dependencies in 10 or so new build files.
True.

`Log4j-api` is used in pretty much every submodule (due to the fine amount of Besu logging) and could be considered a kind of common dependency 🤔 
Being unique in that respect, I can see the value in having as an `api` dependency 👍